contracts/StaderConfig.sol

```diff
@@ -297,6 +297,27 @@ contract StaderConfig is IStaderConfig, AccessControlUpgradeable {
         setContract(SD_INCENTIVE_CONTROLLER, _sdIncentiveController);
     }
 
+    // Access Control
+    function giveCallPermission(
+        address contractAddress,
+        string calldata functionSig,
+        address accountToPermit
+    ) external onlyRole(MANAGER) {
+        bytes32 role = keccak256(abi.encodePacked(contractAddress, functionSig));
+        _grantRole(role, accountToPermit);
```

Comment:
@blockgroot I believe `MANAGER` would not be able to grantRoles ? only `DEFAULT_ADMIN_ROLE` has permissions to grantRole ?

Reply:
ah, understood, you are using internal `_grantRole`